.github/workflows/builds.yml

```diff
@@ -178,9 +178,17 @@ jobs:
       TERM: xterm-256color
 
     steps:
+      - name: Install Qt
+        uses: jurplel/install-qt-action@v3
+        with:
+          dir: /Users/runner
+          version: 6.6
+          modules: qt5compat
+          cache: true
+          cache-key-prefix: macOS-${{matrix.os}}-Qt6.6
+
```

Comment:
This looks OK. Although I can see opposition to this, utilising GitHub Actions' ecosystem is logical.